backend/src/gh.rs

```diff
@@ -161,3 +171,239 @@ fn gen_jwt_token(client_id: &str) -> Result<String> {
         &EncodingKey::from_rsa_pem(&private_key)?,
     )?)
 }
+
+/// Extract the repository name from the `REPO_URL` environment variable in the format `<owner>/<repo>`.
+///
+/// The `REPO_URL` must follow the pattern `https://<host>/<owner>/<repo>.git`. This function removes
+/// the `.git` suffix and parses out the owner and repository name from the URL.
+///
+/// # Parameters
+///  - `repo_url`: A string slice representing the repository URL.
+///
+/// # Returns
+/// A `Result` with a `String` in the format `<owner>/<repo>`, or an error if the `REPO_URL`
+/// environment variable is not set or does not match the expected format.
+///
+/// # Errors
+/// Returns an error if:
+/// - The `REPO_URL` environment variable is not set.
+/// - The format of `REPO_URL` is invalid (does not contain an `<owner>/<repo>` structure).
```

Comment:
This documentation is inaccurate, the function does not make use of environment variables.